perf/perf_uprobe.py

```diff
@@ -94,7 +95,7 @@ def test_uprobe_return(self):
         if 'Added new event' not in output.stderr.decode("utf-8"):
             self.fail("perf: probe of uprobe return failed")
         # RHEL
-        if self.distro_name == "rhel":
+        if self.distro_name == "rhel" and self.distro_version > "7":
```

Comment:
it can be just like `if self.distro_name == "rhel" and detected_distro.distro_version > "7"`